update_compilers/install_libraries.sh

```diff
@@ -69,10 +69,10 @@ update_boost_archive() {
 #
 # See: https://github.com/mattgodbolt/compiler-explorer/issues/1771
 
-install_boost 1.64.0 1.65.0 1.66.0 1.67.0 1.68.0 1.69.0 1.70.0 1.71.0 1.72.0 1.73.0 1.74.0
-update_boost_archive
+#install_boost 1.64.0 1.65.0 1.66.0 1.67.0 1.68.0 1.69.0 1.70.0 1.71.0 1.72.0 1.73.0 1.74.0
```

Comment:
Can you comment these all back in please?

Reply:
Yes of course we will remove the comments. We apologize for this oversight.